resources/de-amp.js

```diff
@@ -1,57 +1,72 @@
 (function () {
   if (typeof deAmpEnabled !== 'undefined' && deAmpEnabled) {
-    const selector = 'a'
-    const attr = 'jsaction'
-    let timer
+    // Polyfill for requestIdleCallback
+    // https://github.com/pladaria/requestidlecallback-polyfill/blob/master/index.js
+    window.requestIdleCallback =
+    window.requestIdleCallback ||
+    function (callBack) {
```

Comment:
best to use `=>` functions here unless you need `this` (which it looks like you dont)